### PR TITLE
Feature: Make the repo JS-only

### PR DESCRIPTION
**Pull request by josemarinas**
## Description

- Remove the javascript folder and flatten its files
- Review the CI/CD flows
- Check the deployment and scripts' internal paths
- Update the readme’s, examples, etc

Task: [APP-1197](https://aragonassociation.atlassian.net/browse/APP-1197)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder of the package after the [UPCOMING] title and before the latest version.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: .github/workflows/javascript_test.yml [hunk 1]
```diff
@@ -14,12 +14,12 @@ on:
     branches-ignore:
       - develop
     paths:
-      - 'javascript/**'
+      - './**'
```
Comment (brickpop) [inline .github/workflows/javascript_test.yml]: Do we need this?

File: .github/labeler.yml [hunk 2]
```diff
@@ -1,15 +1,15 @@
 javascript:
-- javascript/**/*
-- javascript/*
+- ./**/*
+- ./*
```
Comment (brickpop) [inline .github/labeler.yml]: Do we need this tag?

File: .github/labeler.yml [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+- ./modules/common/**/*
+- ./modules/common/*
 
 javascript-client:
-- javascript/modules/client/**/*
-- javascript/modules/client/*
+- ./modules/client/**/*
+- ./modules/client/*
 
 javascript-ipfs:
```
Comment (brickpop) [inline .github/labeler.yml]: Couldn't they be refactored into `common`, `client` and `ipfs` ?

File: .github/workflows/documentation-update.yml [hunk 4]
```diff
@@ -21,10 +21,10 @@ jobs:
           node-version: 16
       - name: Install dependencies
         run: yarn install --frozen-lockfile
-        working-directory: javascript/
+        working-directory: ./
       - name: Generate Docs (Javascript)
         run: yarn run docs:client
-        working-directory: javascript/
+        working-directory: ./
```
Comment (brickpop) [inline .github/workflows/documentation-update.yml]: Needed?

File: .github/workflows/javascript_publish.yml [hunk 5]
```diff
@@ -55,16 +55,16 @@ jobs:
           registry-url: "https://registry.npmjs.org"
 
       - name: install deps
-        working-directory: javascript/
+        working-directory: ./
         run: yarn install --frozen-lockfile
 
       - name: Build subpackages
-        working-directory: javascript/
+        working-directory: ./
```
Comment (brickpop) [inline .github/workflows/javascript_publish.yml]: Needed?

File: .github/workflows/javascript_test.yml [hunk 6]
```diff
@@ -14,12 +14,12 @@ on:
...
       - '.github/workflows/javascript_*'
 
 defaults:
   run:
-    working-directory: './javascript'
+    working-directory: './'
```
Comment (brickpop) [inline .github/workflows/javascript_test.yml]: Needed?

File: javascript/.vscode/launch.json [hunk 7]
```diff
@@ -1,41 +0,0 @@
-{
-  // Use IntelliSense to learn about possible attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
-  "configurations": [
```
Comment (brickpop) [inline javascript/.vscode/launch.json]: Why don't we keep this file in the root path?

Comment (josemarinas) [reply javascript/.vscode/launch.json]: Didn't notice that in the root path, this was in the `.gitignore`

**Review by brickpop — APPROVED** [review]

**Pull request merged**